chaste_codegen/tests/test_codegen.py

```diff
@@ -221,19 +223,42 @@ def test_Opt(tmp_path, model):
                                                model['expected_cpp_path'])
 
 
+def test_missing_V(tmp_path):
+    tmp_path = str(tmp_path)
+    LOGGER.info('Testing missing capacitance\n')
+    model_file = \
+        os.path.join(cg.DATA_DIR, 'tests', 'cellml', 'hodgkin_huxley_squid_axon_model_1952_modified.cellml')
+    chaste_model = cellmlmanip.load_model(model_file)
+
+    # Remove membrane_voltage metadata tag
+    voltage = chaste_model.get_variable_by_ontology_term((OXMETA, 'membrane_voltage'))
+    chaste_model.rdf.remove((voltage.rdf_identity, None, None))
+
+    with pytest.raises(KeyError) as error:
+        chaste_model = cg.NormalChasteModel(chaste_model,
+                                            'hodgkin_huxley_squid_axon_model_1952_modified',
+                                            class_name='hodgkin_huxley_squid_axon_model_1952_modified')
+    assert str(error.value) == \
+        '\'Voltage not tagged in the model!\''
```

Comment:
You can use the `match=` argument to `pytest.raises` to avoid this separate test.

Reply:
ahh thanks!